server/views/layout.pug

```diff
@@ -33,20 +33,26 @@ html(lang='en')
               collaborators: 'Collaborators',
               screenshot: 'Screenshot'
             },
-            lighthousekeys: [ 'seo', 'best-practices', 'accessibility', 'pwa', 'performance']
-          }
+            lighthousekeys: [ 'seo', 'best-practices', 'accessibility', 'pwa', 'performance'],
+            teamsInterval: ''
+          } 
         },
         mounted: function() {
           var self = this;
           self.log();
           if (self.teams && self.teams !== '' && self.teams !== ['']) {
             self.getTeams();
```

Comment:
This was key. In a previous commit, I had removed it and replaced it with startTeamsPolling() below it.